R/oauth-init.R

```diff
@@ -7,10 +7,13 @@
 #'    \code{\link{oauth_app}}
 #' @param permission optional, a string of permissions to ask for.
 #' @param is_interactive Is the current environment interactive?
+#' @param host ip address for the listener
```

Comment:
Thanks for these, will do

On Mon, May 4, 2015, 11:41 AM Hadley Wickham <notifications@github.com>
wrote:

> In R/oauth-init.R
> <https://github.com/hadley/httr/pull/211#discussion_r29610720>:
>
> > @@ -7,10 +7,13 @@
> >  #'    \code{\link{oauth_app}}
> >  #' @param permission optional, a string of permissions to ask for.
> >  #' @param is_interactive Is the current environment interactive?
> > +#' @param host ip address for the listener
>
> @inheritParams ouath_listener would be a bit DRYer
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hadley/httr/pull/211/files#r29610720>.
>